.ci/Jenkinsfile

```
@@ -86,6 +86,20 @@ pipeline {
         }
       }
     }
+    /**
+     Execute integration tests.
+     */
+    stage('TestIntegration') {
+      steps {
+        deleteDir()
+        unstash 'source'
+        dir("${BASE_DIR}"){
+          insideGo(){
+            sh(label: 'Runs the (integration) tests',script: 'mage -debug testIntegration ')
+          }
```

for the moment use this script to run the test instead of indideGo function

```suggestion
          sh(label: 'Runs the (integration) tests',script: '''
            eval "$(gvm 1.14.2)"
            go get -u github.com/magefile/mage
            go get -u github.com/elastic/go-licenser
            go get -u -v golang.org/x/tools/cmd/goimports
            mage -debug testIntegration
          ''')
```